src/managers/NetworkRequestManager.ts

```diff
@@ -168,23 +184,41 @@ class NetworkRequestManager {
 		script: string | undefined,
 		requestId: string,
 		response?: EndpointResponse | undefined,
+		auditInfo?: {
+			log: AuditLog;
+			scriptType: Exclude<RequestEvent['eventType'], 'request'>;
+			associatedId: string;
+		},
 	): Promise<string | undefined> {
 		if (script && script != '') {
```

Comment:
Since you're type coercing to falsy in the first part of the if statement, doesn't it include the second by default (since you're also type coercing that?)

Reply:
yup, empty string is falsey - good call